content/jobs.md

```diff
@@ -51,12 +53,17 @@ Don't see a position that fits your skills? Join our [Slack community](https://p
 
 ---
 
+### Posting Jobs on the PyBaMM Jobs Board
+
 Do you have an open position in battery modeling or simulation? By posting in the PyBaMM Jobs Board, you can:
+
 - Reach a targeted audience of battery modeling experts and enthusiasts
 - Gain access to a pool of candidates with specialized skills in PyBaMM and battery simulation
 - Increase visibility within the PyBaMM community and broader battery modeling ecosystem
 - Support the open-source PyBaMM project and its community
 
 Jobs can be posted by companies, research institutions, or individuals looking to hire for positions related to battery modeling and simulation. All fees collected for job postings will be used to support the development and maintenance of PyBaMM.
 
+For details, please review our [Terms & Conditions](/jobstos/).
```

Comment:
Could we add an admonition here so that this sentence is more prominent? The "Important" one, maybe: https://theme.scientific-python.org/shortcodes/#admonition

Reply:
Sure, thanks :)